### PR TITLE
Update post-production and testing

### PR DESCRIPTION
**Pull request by jamesgk**
This makes various small changes to the touchup and testing scripts, mostly to a) respond to upstream library changes and b) respond to concerns about metadata.

Relates to #109, #142

### CODE REVIEW

**Comment by dougfelt** [comment]
LGTM

File: scripts/build-v2.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def condenseFont(font, scale=.8, stemWidth=185):
 proj.generateFont(Mix([rg, bd], RPoint(0.73, 0.73)),
                   "%s/Bold/Bold/Rg"%FAMILYNAME)
 proj.generateFont(Mix([rg, bd], RPoint(1.125, 1.0)),
-                  "%s/Black/Bold/Bk"%FAMILYNAME)
+                  "%s/Black/Regular/Bk"%FAMILYNAME)
```
Comment (roozbehp) [inline scripts/build-v2.py]: I don't under this change, and the one below it.

File: scripts/run_general_tests.py [hunk 2]
```diff
@@ -60,6 +61,20 @@ class TestMetaInfo(font_tests.TestMetaInfo):
...
 
+class TestNames(font_tests.TestNames):
+    """Bugs:
+    https://github.com/google/roboto/issues/37
+    """
+
+    loaded_fonts = FONTS
+    family_name = 'Roboto'
+    mark_heavier_as_bold = False
+    expected_copyright = 'Font data copyright Google 2015'
```
Comment (roozbehp) [inline scripts/run_general_tests.py]: That is not the expected copyright line. It should be:
Copyright 2011 Google Inc. All Rights Reserved.

Where the year is the first year Roboto was released.

File: scripts/build-v2.py (hunk 1 above)
Comment (jamesgk) [inline scripts/build-v2.py]: This is the way we designate instance metadata; these strings are handled in [instanceNames.py](https://github.com/google/roboto/blob/master/scripts/lib/fontbuild/instanceNames.py#L32). Essentially, this is just saying that the Black weight should be considered a regular (non-bold) font. It is then changed to bold in the Android touchup script.

Comment (roozbehp) [inline scripts/build-v2.py]: Ok. Makes sense.

**Pull request merged**